fhevm/operators_arithmetic_gas.go

```diff
@@ -2,6 +2,59 @@ package fhevm
 
 import "encoding/hex"
 
+func sgxAddSubRequiredGas(environment EVMEnvironment, input []byte) uint64 {
+	input = input[:minInt(65, len(input))]
+
+	logger := environment.GetLogger()
+
+	var lhs, rhs *verifiedCiphertext
+	lhs, rhs, err := get2VerifiedOperands(environment, input)
+	if err != nil {
+		logger.Error("sgxAdd/Sub RequiredGas() ciphertext inputs not verified", "err", err, "input", hex.EncodeToString(input))
+		return 0
+	}
+	if lhs.fheUintType() != rhs.fheUintType() {
+		logger.Error("sgxAdd/Sub RequiredGas() operand type mismatch", "lhs", lhs.fheUintType(), "rhs", rhs.fheUintType())
+		return 0
+	}
+
+	return environment.FhevmParams().GasCosts.FheAddSub[lhs.fheUintType()]
```

Comment:
We shouldn't use FhevmParams, but create a new SgxParams with one field `GasCosts`. Maybe the GasCosts of this struct can be for now 100x less than FHE gas costs